index.html

```diff
@@ -21,11 +23,12 @@
           </section>
      </nav>
```

Comment:
- [ ] Kindly use `header` tag for the container of the "Toolbar" section. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

- [ ] Please make sure the hamburger button is wrap in `nav` tag for semantics reason.